traffic-policy/actions/basic-auth/behavior.mdx

```diff
@@ -0,0 +1,26 @@
+### Behavior
+
+The **basic-auth** action requires incoming HTTP requests to successfully authenticate
+with HTTP Basic Authentication as per RFC 7235. When a request is received, the action verifies
+the request by validating against a known set of `user:password` credentials. If the verification
+is successful, the action allows the request to continue through the action chain and finally
+to your application; if verification fails, the request will be terminated with a
+`401 Unauthorized` response.
+
+#### Additional Restrictions
+
+You can specify only a limited number of `username:password` pairs. The password must be at least
+8 characters and no more than 128 characters. The username can neither be empty nor can it exceed
```

Comment:
The third and second sentence say the same thing differently. Could probably remove the third sentence.

Reply:
ooh nice catch